dist/mvp.less

```
@@ -0,0 +1,473 @@
+@import url(variables);
+
+/* Layout */
+article aside {
+  background: var(--color-secondary-accent);
+  border-left: 4px solid var(--color-secondary);
```

```suggestion
  background: @color-secondary-accent;
  border-left: 4px solid @color-secondary;
```

Использование `var(--color-secondary-accent)` - это использование переменных в чистом CSS